presto-product-tests/README.md

```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+We recommend at least 4GB for such profiles. You can increase docker memory by going to Docker
```

Comment:
... `./presto-product-tests/bin/run_on_docker.sh singlenode -g tpcds -x quarantine` not passing at 6 GB (presto killed), seems working at 7 GB